Various changes

MagisterAdamus opened the pull request:
One note: Shaw seems to have gotten the Italian name of the opera that Higgins sings to himself in Act IV wrong. He calls it `La Fanciulla del Golden West`, but [according to Wikipedia](https://en.wikipedia.org/wiki/La_fanciulla_del_West), there's no `Golden` in the original Italian-language title, so it's actually just `La Fanciulla del West`. Should this be corrected?

MagisterAdamus committed:
Fix Wikipedia link

MagisterAdamus committed:
Restore accents to French-language expression "Théâtre Français"

HathiTrust scans reference (p. 113): https://babel.hathitrust.org/cgi/pt?id=mdp.39015030702586&seq=247

MagisterAdamus committed:
Manually add additional semantics to the Preface

MagisterAdamus committed:
Act I: Fix typo

HathiTrust scans reference: https://babel.hathitrust.org/cgi/pt?id=mdp.39015030702586&seq=252

MagisterAdamus committed:
Act I: Fix typo

HathiTrust scans reference: https://babel.hathitrust.org/cgi/pt?id=mdp.39015030702586&seq=255

MagisterAdamus committed:
Act II: Add missing persona markup to stage directions

MagisterAdamus committed:
Act II: Add missing space before stage direction

MagisterAdamus committed:
[Editorial] visible Speech -> Visible Speech

"Visible Speech" is the name of a phonetic transcription system: https://en.wikipedia.org/wiki/Visible_Speech

MagisterAdamus committed:
[Editorial] flower-shop -> flower shop

This change is for the sake of consistency throughout the text.

MagisterAdamus committed:
Act II: Fix typo

HathiTrust scans reference: https://babel.hathitrust.org/cgi/pt?id=mdp.39015030702586&seq=280

MagisterAdamus committed:
[Editorial] anyway -> any way

MagisterAdamus committed:
Act V: Add semantics to given-name initial

MagisterAdamus committed:
Act V: Add missing persona markup to stage direction

MagisterAdamus committed:
Mark up Italian-language opera name as Italian

MagisterAdamus committed:
[Editorial] may be -> maybe

MagisterAdamus committed:
[Editorial] em -> ’em

MagisterAdamus committed:
Italicize phoneme and convert it to sentence-case

MagisterAdamus committed:
Act IV: Add missing persona markup to stage direction

MagisterAdamus committed:
Act III: Add missing persona markup to stage directions

MagisterAdamus committed:
[Editorial] til -> ’til

acabal commented:
Great, thanks. Yes, since it's in a stage direction you can correct the title of the play.

MagisterAdamus commented:
> Great, thanks. Yes, since it's in a stage direction you can correct the title of the play.

You're welcome! Just finished reading it now that I'm finally able to read plays on my Kobo without the text being cut off on every page. I'll make this change and push to this branch.

MagisterAdamus committed:
[Editorial] Correct Italian-language name of opera in stage direction

MagisterAdamus commented:
Oh wait, don't merge this yet! I just found one place where there was missing emphasis.

MagisterAdamus committed:
Act V: Restore missing emphasis/italics

HathiTrust scans reference: https://babel.hathitrust.org/cgi/pt?id=mdp.39015030702586&seq=336

MagisterAdamus commented:
I think there might still be some cases of missing emphasis, just as a note to anyone who works on this later. The scans indicate emphasis by adding spaces between the letters of an emphasized word, instead of using italics, with the exception of the personal pronoun `I` (since it's impossible to space out a single-letter word). So the word `I`, when emphasized, is set in italics.

MagisterAdamus committed:
Act V: Do not format "translation" of Eliza's dialectal pronunciation as a stage direction

HathiTrust scans reference: https://babel.hathitrust.org/cgi/pt?id=mdp.39015030702586&seq=337

MagisterAdamus commented:
Re: the last commit (i.e., commit `86245cd`), it was incorrectly formatted as a stage direction, but instead it's a note to the reader translating Eliza's former low-class dialectal pronunciation into Standard English orthography. It could be converted into an endnote, but it definitely shouldn't be formatted as a stage direction.

acabal commented:
Since you're looking at this, can you restore emphasis, and convert that translation to an endnote?

MagisterAdamus commented:
> Since you're looking at this, can you restore emphasis, and convert that translation to an endnote?

Okay, I'll go through the scans page by page tomorrow and see if there's any missing emphasis. Then I'll do the endnote conversion.

MagisterAdamus committed:
Act I: Add missing emphasis

MagisterAdamus committed:
Act II: Restore missing colon

HathiTrust scans reference (p. 138): https://babel.hathitrust.org/cgi/pt?id=mdp.39015030702586&seq=272

MagisterAdamus committed:
Act II: Add missing emphasis

MagisterAdamus committed:
Act II: Restore original dialectal spelling

HathiTrust scans reference (p. 146): https://babel.hathitrust.org/cgi/pt?id=mdp.39015030702586&seq=280

MagisterAdamus committed:
Act III: Add missing emphasis

MagisterAdamus committed:
Act IV: Add missing emphasis

MagisterAdamus committed:
Act V: Add missing emphasis

MagisterAdamus committed:
Act V: Restore missing em-dash

HathiTrust scans reference (p. 199): https://babel.hathitrust.org/cgi/pt?id=mdp.39015030702586&seq=333

MagisterAdamus commented:
OK, I've gone through the entire work, line by line, from the preface through to the concluding essay which comes after the final act and restored all the missing emphasis that I found. I've also corrected any other mistakes that I found.

I still have to convert the note to the reader into an endnote.

MagisterAdamus committed:
[Editorial] Move note to reader in Act V into endnotes

MagisterAdamus committed:
Update manifest and spine

MagisterAdamus committed:
Update ToC

MagisterAdamus committed:
Update content.opf to list author as annotator

MagisterAdamus commented:
OK, I've done the endnote conversion. `se lint` only complains that the word count is incorrect, but I don't know how to fix this.

acabal commented:
Excellent work, thanks!

MagisterAdamus commented:
Thank you!
